org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java

```diff
@@ -1612,13 +1618,20 @@ private ReportEngine runReport(int AD_Process_ID, int AD_PrintFormat_ID, String
 				return null;
 			//
 			ProcessInfo pi = new ProcessInfo (process.getName(), process.getAD_Process_ID(), 0, 0);
+			if(process.getJasperReport() != null) {
+				pi.setExport(true);
+				pi.setExportFileExtension("html");
+			}
 			pi.setAD_User_ID(Env.getAD_User_ID(Env.getCtx()));
 			pi.setAD_Client_ID(Env.getAD_Client_ID(Env.getCtx()));
 			pi.setAD_PInstance_ID(pInstance.getAD_PInstance_ID());		
 			if (!process.processIt(pi, null) && pi.getClassName() != null) 
 				throw new IllegalStateException("Process failed: (" + pi.getClassName() + ") " + pi.getSummary());
 		
 			//	Report
+			if(process.getJasperReport() != null) {
+				ServerProcessCtl.process(pi, null);
+			}
```

Comment:
this part seems wrong, you either have JasperReport or ReportEngine, you can't have both here.

Reply:
i see. it's my first approach when trying to run jasper report and then i think i need to create separated method so i create runJasperReport method